prod/westeurope/linux/appbackendl1/web_test/ping.xml

```diff
@@ -0,0 +1,6 @@
+<WebTest Name="${name}" Id="ABD48585-0831-40CB-9069-682EA6BB3583" Enabled="True" CssProjectStructure="" CssIteration="" Timeout="${timeout}" WorkItemIds=""
+    xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010" Description="" CredentialUserName="" CredentialPassword="" PreAuthenticate="True" Proxy="default" StopOnError="False" RecordedResultFile="" ResultsLocale="">
+    <Items>
+        <Request Method="${http_method}" Guid="a5f10126-e4cd-570d-961c-cea43999a200" Version="1.1" Url="${url}" ThinkTime="0" Timeout="${timeout}" ParseDependentRequests="False" FollowRedirects="True" RecordResult="True" Cache="False" ResponseTimeGoal="0" Encoding="utf-8" ExpectedHttpStatusCode="${http_status_code}" ExpectedResponseUrl="" ReportingName="" IgnoreHttpStatusCode="False" />
+    </Items>
+</WebTest>
```

Comment:
`ping.xml` is not required. It will be used default `ping.xml` defined in webtest module

Reply:
it's not going to work .... I removed the xml file, ran the terragrunt init, terragrunt plan and I am getting this error:

```
Error: Invalid function argument

  on main.tf line 18, in resource "azurerm_application_insights_web_test" "application_insights_web_test":
  18:   configuration = templatefile(var.configuration, { name = local.resource_name,
  19:     url         = var.url,
  20:     timeout     = var.timeout,
  21:     http_method = var.http_method,
  22:   http_status_code = var.http_status_code, })
    |----------------
    | var.configuration is "ping.xml"

Invalid value for "path" parameter: no file exists at ping.xml; this function
works only with files that are distributed as part of the configuration source
code, so if this file will be created by a resource in this configuration you
must instead obtain this result from an attribute of that resource.

Releasing state lock. This may take a few moments...
[terragrunt] 2021/02/03 10:11:56 Hit multiple errors:
exit status 1
```

Reply:
Strange I tried a plan without errors... I'll retry a plan and an apply on appbackendl1

Reply:
I successfully applied without errors. I only removed `ping.xml` file